skills/findChannel.js

```diff
@@ -0,0 +1,21 @@
+class Project{
+  constructor(channel){
+    this.channel_id = channel;
+  }
+
+  static findByChannel(channel){
+    return new Promise(function(resolve,reject){
+      let p = new Project(channel)
```

Comment:
Single letter variables don't provide much for the reader. You're writing for future you and future developers who know nothing of your intent. Make it clear.

Reply:
ok, will do.